### DISCUSSION TITLE
How do you override nvim-colorizer settings?

### DISCUSSION THREAD

**Discussion by bendavis78** [NvChad/NvChad#2373] [Q&A] [answered]
I'd like to disable highlighting of color names (for example, the word "Azure" is always highlighted). How do you override settings of default plugins?

I tried putting it in my plugins config:

```  
  {"NvChad/nvim-colorizer", opts={names = false}},
```

But Lazy shows the following error:
```
 Failed (1)
     nvim-colorizer 
        Cloning into '/Users/bdavis/.local/share/nvim/lazy/nvim-colorizer'...
        fatal: could not read Password for 'https://ghp_jrssu4iTFH...5r3f80LX@github.com': terminal prompts disabled
```

Not sure why the git remote is configured with that username. Is that normal?

**Comment by siduck** [comment, marked as answer]
![image](https://github.com/NvChad/NvChad/assets/59060246/09afbc44-7d7b-4378-904f-250abc5054c1)
![image](https://github.com/NvChad/NvChad/assets/59060246/7fa2b199-718c-4a25-acd2-8f383ce437b1)

so directly it wont work in opts, you should modify proper tables

**Reply by bendavis78** [reply]
Thanks, I saw those options but where do I put them in my nvchad config? What do you mean by "proper tables"?

**Reply by siduck** [reply]
@bendavis78 i'll give a hint

```lua
  {
      "NvChad/nvim-colorizer",
      opts = {
        filestypes = blabla, 
      }
   },
```

**Reply by bendavis78** [reply]
I finally figured out what was going on -- I had the repo wrong, it was missing `.lua` at the end. I forgot git clone won't tell you if a repo is not found. 

In case any other lost soles ends up here, here's the config I used:

```lua
  -- Override colorizer options
  {"NvChad/nvim-colorizer.lua", opts={
    user_default_options = {
      names = false,
    },
  }},
```
